example-hooks/package.json

```diff
@@ -48,11 +48,11 @@
     "babel-preset-stage-0": "^6.5.0",
     "babel-runtime": "^6.26.0",
     "csstype": "^2.5.6",
-    "fork-ts-checker-webpack-plugin": "^0.3.0",
+    "fork-ts-checker-webpack-plugin": "^3.1.1",
     "patch-package": "^6.2.0",
     "postinstall-postinstall": "^2.0.0",
     "relay-compiler": "^7.0.0",
-    "relay-compiler-language-typescript": "file:.yalc/relay-compiler-language-typescript",
+    "relay-compiler-language-typescript": "file:../lib",
```

Comment:
Why is this being included in this PR?

Reply:
My bad... was just trying to make sure things still worked, and not using yalc, this was the fix that allowed that to happen. Must have snuck in.

Did you want me to remove this from the PR?

Reply:
Yes please 🙏